includes/class-social.php

```diff
@@ -56,7 +56,7 @@ public function form( $instance ) {
 
 		foreach ( $fields as $key => $field ) {
 
-			if ( ! isset( $field['social'] ) ) {
+			if ( ! isset( $field['social'] ) || $this->is_social_media_deprecated( $field ) ) {
```

Comment:
@EvanHerman Oh no sorry, you are right. This is for the icons, I was was mistaken and thought it was for the fields.